examples/todomvc/test/todomvc/runner.cljs

```diff
@@ -0,0 +1,10 @@
+(ns todomvc.test.runner
```

Comment:
These tests should be run as part of the CircleCI run. I can help with this if you want?